src/main/resources/application.yml

```diff
@@ -44,6 +44,7 @@ app:
   conceptTreeFile: ${CONCEPT_TREE_FILE:ontology/conceptTree.json}
   fhirTranslationEnabled: ${FHIR_TRANSLATE_ENABLED:false}
   cqlTranslationEnabled: ${CQL_TRANSLATE_ENABLED:true}
+  apiBaseUrl: ${API_BASE_URL:}
```

Comment:
Please also add this to the `README.md` file alongside an explanation on what this does.